lib/model/query/datasets.js

```diff
@@ -121,9 +121,14 @@ const _getByIdSql = ((fields, datasetId) => sql`
 
 const _getByNameSql = ((fields, datasetName, projectId, includeForms) => sql`
     SELECT
-        ${fields}
+        ${fields}, stats."lastEntity" "datasets!lastEntity"
     FROM
         datasets
+    LEFT OUTER JOIN (
+      SELECT "datasetId", MAX(COALESCE("updatedAt", "createdAt")) "lastEntity" 
```

Comment:
`coalesce` always returns a single value (first non-null), right? So this would always call `max` on a single value and return that?